### ISSUE TITLE
Version key not present in schema.json

### ISSUE THREAD

**Issue by AaronFriel** [pulumi/pulumi-eks#688]

**Comment by stack72** [comment]
@AaronFriel how can we verify this is working when we add it?

**Comment by AaronFriel** [comment]
I don't think this is breaking anything, just out of spec with the Pulumi Package Schema doc. Don't think we have any tests to verify this.
